src/pages/authentication/Signup.tsx

```diff
@@ -1,3 +1,113 @@
+import { Link } from 'react-router-dom';
+
 export function SignupPage() {
-  return <div>SignupPage</div>;
+  return (
+    <div className='SignupForm'>
+      <div className='max-w-screen-xl px-4 py-16 mx-auto sm:px-6 lg:px-8'>
+        <div className='max-w-lg mx-auto'>
+          <h1 className='text-2xl font-bold text-center text-blue-800 sm:text-3xl'>
+            Join Opencampus to get hired by top MNCs
+          </h1>
+
+          <p className='max-w-md mx-auto mt-4 text-center text-gray-500'>
+            an integrated campus hiring and training automation platform
+          </p>
+
+          <form
+            action=''
+            className='p-8 mt-6 mb-0 space-y-4 rounded-lg shadow-2xl'
+          >
+            <p className='text-lg font-medium'>
+              Create your Opencampus account today!
+            </p>
+            {/* write your name here, don't be shy */}
+            <div className='relative mt-1'>
+              <label className='text-sm font-medium'>First Name</label>
+              <div className='relative mt-1'>
+                <input
+                  type='text'
+                  id='firstname'
+                  className='w-full p-4 pr-12 text-sm border-gray-200 rounded-lg shadow-sm'
+                  placeholder='Enter First Name'
+                />
+              </div>
```

Comment:
Add for attribute in label tag referencing the id of the input field.


```suggestion
              <label className='text-sm font-medium' for='firstname'>First Name</label>
              <div className='relative mt-1'>
                <input
                  type='text'
                  id='firstname'
                  className='w-full p-4 pr-12 text-sm border-gray-200 rounded-lg shadow-sm'
                  placeholder='Enter First Name'
                />
              </div>
```

Do this for all labels.